### PR TITLE
Update middleware lib specs to 0.8.0-alpha.1 version

### PR DESCRIPTION
**Pull request by vnavarro**
Since ios_release folder is necessary and its only present from 0.8.0-alpha.1 onward we needed to update the specs in order to dependant projects to work.

### CODE REVIEW

**Comment by vnavarro** [comment]
@mmassaki can you check this?

File: Specs/MiddlewareLib/0.8.0/MiddlewareLib.podspec.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
   },
   "authors": {"Taqtile": "contato@taqtile.com"},
   "source": {
-    "git": "https://github.com/indigotech/br-npc-middleware-lib-java/",
-    "tag": "0.8.0"
+    "git": "git@github.com:indigotech/br-npc-middleware-lib-java.git",
```
Comment (mmassaki) [inline Specs/MiddlewareLib/0.8.0/MiddlewareLib.podspec.json]: @vnavarro 
why did you change to SSH URL now?
It's better to use HTTPS instead.

**Comment by mmassaki** [comment]
@vnavarro 
you should rename the version directory as well to use this new version.

**Comment by vnavarro** [comment]
@mmassaki done!

**Comment by mmassaki** [comment]
:+1:
